### DISCUSSION TITLE
How to pass initialValues to Date Range Input in form (error?)

### DISCUSSION THREAD

**Discussion by mateuscqueiros** [ant-design/ant-design#51113] [Q&A] [answered]
How to pass initialValues to rangeDate input in a form? I've tried the following ("rangeDates" is the name of my form item), but when i open the form it throws an error `date.isValid is not a function` .

```tsx
 {
  ...formInitialValues,
  rangeDates: [
    dayjs(project?.startDate, "DD-MM-YYYY"),
    dayjs(project?.endDate, "DD-MM-YYYY"),
  ],
}

```

Am I doing something wrong or is it an error?

**Comment by kiner-tang** [comment, marked as answer]
Why do you pass the format in initValues? I think you should specify the date presentation format in DateRangePicker by format.
Just like this:
https://stackblitz.com/edit/react-krhy4u?file=demo.tsx
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/54c1e100-16ae-47fb-b796-0795268a03d6">

**Reply by mateuscqueiros** [reply]
Hello @kiner-tang! I think that may be it. Thanks for the help!

Also thanks for the sandbox example, I didn't know about the `type` option in the rules.
